torch/_torch_docs.py

```diff
@@ -3034,15 +3034,17 @@ def merge_dicts(*dicts):
 
 Creates a :class:`Tensor` from a :class:`numpy.ndarray`.
 
-The returned tensor and :attr:`ndarray` share the same memory. Modifications to
-the tensor will be reflected in the :attr:`ndarray` and vice versa. The returned
-tensor is not resizable.
-
 It currently accepts :attr:`ndarray` with dtypes of ``numpy.float64``,
 ``numpy.float32``, ``numpy.float16``, ``numpy.complex64``, ``numpy.complex128``,
 ``numpy.int64``, ``numpy.int32``, ``numpy.int16``, ``numpy.int8``, ``numpy.uint8``,
 and ``numpy.bool``.
 
+.. warning::
+    The returned tensor and :attr:`ndarray` share the same memory. Modifications to
+    the tensor will be reflected in the :attr:`ndarray` and vice versa. 
+    Writing to a tensor created from a read-only NumPy array is not supported and 
```

Comment:
Sorry what I said before was confusing. I meant just this one sentence:

"Writing to a tensor created from a read-only NumPy array is not supported and will result in undefined behavior." 

should be a warning, not the entire paragraph. The rest of the paragraph can go back where it was.